perception/autoware_camera_streampetr/config/tensorrt_stream_petr.param.yaml

```diff
@@ -0,0 +1,19 @@
+/**:
+  ros__parameters:
+    # see https://github.com/open-mmlab/mmdetection3d/blob/fe25f7a51d36e3702f961e198894580d83c4387b/mmdet3d/evaluation/metrics/nuscenes_metric.py#L386-L393
+    rois_number: 6
+    model_params:
+      class_names: ["CAR", "TRUCK", "BUS", "BICYCLE", "PEDESTRIAN"]
+      fp16_mode: true
+      use_temporal: true
+      detection_range: [-61.2, -61.2, -10.0, 61.2, 61.2, 10.0]
+      input_image_height: 480
+      input_image_width: 640
+      num_proposals: 5400
+      max_camera_time_diff: 0.15                   # Maximum permitted time difference between camera images
```

Comment:
Just for my understandings.
Are there any parameters already included in onnx file description?
I mean if some of the parameters are highly depends on onnx file, those parameter should be provided with model file.